packages/react-magma-dom/src/components/Stepper/Stepper.test.js

```diff
@@ -13,35 +13,97 @@ const testId = 'test-id';
 
 describe('Stepper', () => {
   it('should find element by testId', () => {
-    const { getByTestId } = render(<Stepper ariaLabel="progress"  testId={testId}></Stepper>);
+    const { getByTestId } = render(
+      <Stepper ariaLabel="progress" testId={testId}></Stepper>
+    );
 
     expect(getByTestId(testId)).toBeInTheDocument();
   });
 
-  it('Does not violate accessibility standards', () => {
-    const { container } = render(<Stepper ariaLabel="progress" role="form"></Stepper>);
+  describe('Accessibility', () => {
+    it('Does not violate accessibility standards', () => {
+      const { container } = render(
+        <Stepper ariaLabel="progress" role="form"></Stepper>
+      );
 
-    return axe(container.innerHTML).then(result => {
-      return expect(result).toHaveNoViolations();
+      return axe(container.innerHTML).then(result => {
+        return expect(result).toHaveNoViolations();
+      });
     });
-  });
 
-  it('should use the custom aria-label', () => {
-    const { getByLabelText } = render(
-      <Stepper ariaLabel="custom" testId={testId}  currentStep={0}>
-        <Step />
-        <Step />
-      </Stepper>
-    );
+    it('should use the custom aria-label', () => {
+      const { getByLabelText } = render(
+        <Stepper ariaLabel="custom" testId={testId} currentStep={0}>
+          <Step />
+          <Step />
+        </Stepper>
+      );
 
-    const customAriaLabel = getByLabelText('custom', {selector: 'div'});
+      const customAriaLabel = getByLabelText('custom', { selector: 'div' });
+
+      expect(customAriaLabel).toBeInTheDocument();
+    });
+
+    it('should have hidden labels for screen readers when layout is set to "showLabels"', () => {
+      const { getByTestId } = render(
+        <Stepper
+          ariaLabel="progress"
+          layout={StepperLayout.showLabels}
+          currentStep={0}
+        >
+          <Step
+            testId={`${testId}-1-nolabels-hiddenlabeltext`}
+            label={`${TEXT}-1`}
+          />
+          <Step
+            testId={`${testId}-2-nolabels-hiddenlabeltext`}
+            label={`${TEXT}-2`}
+          />
+        </Stepper>
+      );
+
+      expect(
+        getByTestId(`${testId}-1-nolabels-hiddenlabeltext`)
+      ).toHaveTextContent(`Step 1, ${TEXT}-1, Step active`);
+      expect(
+        getByTestId(`${testId}-2-nolabels-hiddenlabeltext`)
+      ).toHaveTextContent(`Step 2, ${TEXT}-2, Step incomplete`);
```

Comment:
From my understanding of 
```
Step 1 of 4. Choose materials. Step completed.
Step 2 of 4. Additional details aria-current="step"
Step 3 of 4. Confirm.
```

if `currentStep={1}`, then 
- step 1 should have text content `Step 1. ${TEXT}-1. Step completed` and aria-current should be "false"
- step 2 should have text content `Step 1. ${TEXT}-2.` and aria-current should be "step"
- step 3 should have text content `Step 1. ${TEXT}-3.` and aria-current should be "false"